pkgs/misc/cups/drivers/carps-cups/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchgit
+, cups
+}:
+
+stdenv.mkDerivation {
+  pname = "carps-cups";
+  version = "18d80d1";
+  src = fetchgit {
```

```suggestion
  version = "unstable-XXXX-XX-XX";
  
  src = fetchgit {
```

Please use the date of the commit.